contact.md

```diff
@@ -28,7 +28,7 @@ if (originalSearchParams.has('body')) {
 	searchParams.set('body', originalSearchParams.get('body'));
 }
 
-element.search = searchParams.toString();
+element.href = `${element.href}?${searchParams.toString()}`;
```

Comment:
I don't like string concat for URLs. You could use `new URL()` to generate the URL instead of URLSearchParams

Reply:
#14